_scicomputing/store_objectstore.md

```diff
@@ -47,16 +47,19 @@ In the event that you need to share a very large dataset (e.g. >5TB) with an ext
 
 S3 (the Simple Storage Service) is an object store very much like the Economy file service, though provided by Amazon Web Services.  Storage resources in S3 are organized much like the other Fred Hutch Object and Block systems, with a "PI bucket" for each investigator at the Hutch which is analogous to the investigator directories available in the traditional file system. A specialized client (the AWS command line interface) is used to upload the data from local storage to S3 storage.
 
-#### Backup and Security
+#### Backup, Retention, and Security
 
 Data on this service is not backed up in the traditional sense, but rather versioned: if a new version of a file is uploaded, the older version is saved in S3.  Similarly, if data is deleted, the versions aren't and can be retrieved for up to 60 days.  The Fred Hutch supported PI buckets in S3 are appropriate for storage of restricted data, including PHI.
 
+All Economy Cloud S3 buckets have Intelligent Tiering enabled. This means that data is automatically archived in long-term storage after it has not been accessed for a certain period of time. This is done to keep costs down for users. All files are still retrievable at any time; however there may be a short turnaround time for files that haven't been accessed in over 90 days. A full description of intelligent tiering and how it works can be found here: https://aws.amazon.com/s3/storage-classes/intelligent-tiering/
```

Comment:
Couple points on this:

- Can we remove the term "archived" for this activity?  I believe that most readers will interpret this to mean that it's moved out of the bucket and into some other location (already happened once).
- I'd suggest that we use the retrieval time (3-5 hours) instead of the "short turnaround" phrase.  Setting expectations on retrieval will be important as time goes on.